e1 done - test passed successful

Gaydarenko opened the pull request:
on practice

Gaydarenko committed:
e1 done - test passed successful

aeksei reviewed (changes requested):
А почему поиск в глубину сделан через очередь? Ведь реализация должна быть через рекурсивные вызовы своих соседей

---

Если стоит статус **Changes requested**, то нужно в данную ветку внести обязательные изменения, которые представлены ниже. Если выше представлены комментарии, то они носят рекомендательный характер.
После нужно сделать "push", тогда коммиты с изменениями сразу попадут в "pull request". 
После этого нужно будет запросить "re-request review"

Gaydarenko committed:
Добавил два решения:
2) внутри функции сделал еще одну функцию для рекурсивного вызова
3) в головную функцию добалил ключ viewed=None. Но при таком решении не понимаю как избавиться от вывода промежуточных результатов

Gaydarenko committed:
Добавил ещё одно решение (как третье, только на списке)
